static/templates/message_edit_form.hbs

```diff
@@ -10,7 +10,7 @@
         <div class="controls edit-controls">
             <div class="message_edit_header">
                 <div class="stream_header_colorblock" {{#unless is_stream_editable}}style="display:none"{{/unless}}></div>
-                <div class="select_stream_dropdown" {{#unless is_stream_editable}}style="display:none"{{/unless}}>
+                <div class="select_stream_setting" {{#unless is_stream_editable}}style="display:none"{{/unless}}>
```

Comment:
I don't think this class is used in the other dropdown?  

```
$ git grep select_stream_setting
static/styles/zulip.css:.select_stream_setting {
static/templates/message_edit_form.hbs:                <div class="select_stream_setting" {{#unless is_stream_editable}}style="display:none"{{/unless}}>
```

I also think perhaps we should call this class `select_stream_widget`?  That seems like the right name for the outer element.

Reply:
`select_stream_setting `  is used move_topic_to_stream popover. The class name is autogenerated, so you won't find it on grep.

Reply:
Oh, fascinating.  OK, probably all we need is a comment.  Though @aryanshridhar we might want to think about avoiding the auto-generated class name, since greppability can be pretty valuable for understanding CSS.

Reply:
I am not sure where the comment will go.

Reply:
Oh, I'll do it in the CSS file -- just want to make sure someone doesn't delete this CSS if at some point they change the message_edit template.